jni/ooni_test_wrapper.hpp

```diff
@@ -0,0 +1,52 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+#ifndef JNI_OONI_TEST_WRAPPER_HPP
+#define JNI_OONI_TEST_WRAPPER_HPP
+
+#include <measurement_kit/ooni.hpp>
+#include <string>
+
+class OoniTestWrapper {
+  public:
+    OoniTestWrapper(std::string test_name) {
+        if (test_name == "tcp_connect") {
+            real_test_.reset(new mk::ooni::TcpConnect);
```

Comment:
@lorenzoPrimi here we should add also the other tests. By the way, throwing an exception is not super optimal here because the exception is not routed by Swig and causes the application to terminate.